<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLevel</class>
 <widget class="QWidget" name="QLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnLev1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLev9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡9</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>QLevel</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
